sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -53,6 +53,16 @@ def __init__(self, hidden_layer_sizes, activation, solver,
                  warm_start, momentum, nesterovs_momentum, early_stopping,
                  validation_fraction, beta_1, beta_2, epsilon):
         self.activation = activation
+        # If a single activation function is given,
+        # use it for each hidden layer
+        if type(self.activation) == str and\
```

Comment:
do not use `\` but put the statement between `()` and you can break the lines then

Reply:
oh ... I missed this one. we are using six for the string type checking

```python
from sklearn.externals.six import string_types

if isinstance(self.activation, string_types) and ...
```

Reply:
done!